### DISCUSSION TITLE
Suggestions for the future

### DISCUSSION THREAD

**Discussion by TaxicabMessiah** [zaps166/QMPlay2#801] [Ideas]
-An indication of download percentage when extracting audio from external sources like Youtube
-Ability to bulk edit files by highlighting all and right clicking into properties.
-Waveform widget option for each song when playing
-Subtitles & audio track toggle buttons for visual media that indicate how many of each track type are available.
-Wallpaper scaling features like 'shrink to fit'
-A button for "randomize music selection" next to the core 'play' and 'pause' ones.
-Ability to modify icons & change fonts.
-Ability to retain the seek bar & play/pause buttons in compact mode as opposed to only popping up on hover
-Ability to prevent the playlist from displaying media history.
-Text in the media information widget to have the wraparound feature.  
-Save/export/load EQ presents 
-When increasing volume slider to 150% or 200% have the area of the slider that goes beyond 100% a different color
-Ability to have Video Filter Adjustment settings (brightness, etc) as a widget - just like the EQ widget

**Comment by zaps166** [comment]
> An indication of download percentage when extracting audio from external sources like Youtube

You can see download progress, conversion is ususally fast-enough.

> Ability to bulk edit files by highlighting all and right clicking into properties.

Nice, but too much work to me.

> Waveform widget option for each song when playing

It's already reported in "Issues".

> Subtitles & audio track toggle buttons for visual media that indicate how many of each track type are available.

Idk if I understand your sentence - you can see it in Information panel and in Playback menu (Audio streams, Subtitles streams, Video streams, Programs),

> Wallpaper scaling features like 'shrink to fit'

It's a bit tricky and I don't want to spend time on wallpaper. Prepare for your size in any image editing software and load.

> A button for "randomize music selection" next to the core 'play' and 'pause' ones.

Should be reported in "Issues" as feature request.

> Ability to modify icons & change fonts.

Should be reported in "Issues" as feature request, but I don't want to spend time on this for now.

> Ability to retain the seek bar & play/pause buttons in compact mode as opposed to only popping up on hover

It's what I wanted to have in compact mode. If you want just seek bar panel you can close all other panels and create profile in Options.

> Ability to prevent the playlist from displaying media history.

"Automatically delete ungrouped entries" in "General settings".

> Text in the media information widget to have the wraparound feature.

I don't understand this.

> Save/export/load EQ presents

You can already save your preset, but you can't export and load -> should be reported in "Issues" as feature request.

> When increasing volume slider to 150% or 200% have the area of the slider that goes beyond 100% a different color

Should be reported in "Issues" as feature request, however I think it's better to use system volume for this - it's available in PulseAudio and PipeWire easily available in Xfce4 and Plasma volume control.

> Ability to have Video Filter Adjustment settings (brightness, etc) as a widget - just like the EQ widget

Isn't Settings->Video filters enough?
